apps/web-giddh/src/app/reports/components/purchase-register-component/purchase.register.component.ts

```diff
@@ -92,7 +92,12 @@ export class PurchaseRegisterComponent implements OnInit {
     }
 
     ngOnInit() {
-
+        this.router.events.pipe(
+            filter(event => (event instanceof NavigationStart && !event.url.includes('/reports'))),
```

Comment:
This is not good to check url
Some time we have to change routing then it will get affected